### PR TITLE
View temp calibration

### PR DESCRIPTION
**Pull request by pianistrevor**
Add menu for viewing temperature calibration offset, implements #225

### CODE REVIEW

File: src/UIState/SeeTempCalOffset.cpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#include "TemperatureCalibration.h"
+
+void SeeTempCalOffset::start() {
+  // First line: prompt
+  LiquidCrystal_TC::instance()->writeLine(prompt(), 0);
+  // Second line: the offset (5 digits of precision)
+  char buffer[17];
+  dtostrf(TempProbe_TC::instance()->getCorrection(), 7, 5, buffer);
+  LiquidCrystal_TC::instance()->writeLine(buffer, 1);
+}
```
Comment (jgfoster) [inline src/UIState/SeeTempCalOffset.cpp:18]: Sometime it would be nice to have a newline at the end of this file.

**Review by jgfoster — APPROVED** [review]

File: src/UIState/SeeTempCalOffset.cpp (hunk 1 above)
Comment (pianistrevor) [reply src/UIState/SeeTempCalOffset.cpp:18]: Will do in the fix commit.

**Review by prestoncarman — CHANGES_REQUESTED** [review]
Is it possible to test this new feature?

**Comment by pianistrevor** [comment]
I'll write test coverage tomorrow, thanks!

**Review by jgfoster — APPROVED** [review]
Looks good. @prestoncarman ?

**Review by prestoncarman — APPROVED** [review]
LGTM

**Pull request merged**